source/connect/in-use-encryption.txt

```diff
@@ -0,0 +1,263 @@
+.. _in-use-encryption-tab:
+
+================================
+In-Use Encryption Connection Tab
+================================
+
+.. default-domain:: mongodb
+
+.. contents:: On this page
+   :local:
+   :backlinks: none
+   :depth: 1
+   :class: singlecol
+
+:guilabel:`In-Use Encryption` is an Enterprise/Atlas only feature. You need a replica set 
+to use this connection option. Your replica set can be a single node or larger. 
+
+The :guilabel:`In-Use Encryption` connection tab allows you to connect your
+deployments using :v6.0:`Queryable Encryption </core/queryable-encryption/>`.
+
+Procedure
+---------
+
+.. procedure::
+   :style: normal
+
+   .. step:: Click :guilabel:`Advanced Connection Options`.
+
+      .. figure:: /images/compass/advanced-connection-options.png
+         :figwidth: 690px
+         :alt: New Advanced Connection Options
+
+   .. step:: Click the :guilabel:`In-Use Encryption` tab.
+
+      .. procedure::
+         :style: connected
+
+         .. step:: Provide a :guilabel:`Key Vault Namespace`.
+
+            A :guilabel:`Key Vault Namespace` refers to a collection that 
+            contains all the data keys used for encryption and decryption. 
+
+            Specify a collection in which data encryption keys are stored in 
+            the format ``<db>.<collection>``. The non-official default 
+            database/collection for keyVault is ``encryption._keyVault``.
```

Comment:
I think a lot of our docs use two underscores, it might be good to be consistent with that:

```suggestion
            database/collection for keyVault is ``encryption.__keyVault``.
```

Reply:
Done.